packages/lib/src/components/Dropin/components/DropinComponent.tsx

```diff
@@ -126,6 +126,12 @@ export class DropinComponent extends Component<DropinComponentProps, DropinCompo
         this.setState({
             showDefaultPaymentMethodList: true
         });
+
+        this.props.modules?.analytics.sendAnalytics('dropin', {
+            type: ANALYTICS_EVENT.info,
+            infoType: InfoEventTypes.clicked,
+            target: 'otherpaymentmethod_button'
```

Comment:
Similar to my comment above - can the backend handle this target value, "otherpaymentmethod_button" without errors?